spec/system/take_part_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe "TakePart" do
+  before do
+    content_store_has_example_item("/government/get-involved/take-part/tp1", schema: :take_part)
+  end
+
+  context "/government/get-involved/take-part/:slug" do
```

Comment:
This is interesting. Should the `context` for system tests be set to the path? Is that needed because it's specified in the `visit` call. 
For other [document types](https://github.com/alphagov/frontend/blob/main/spec/system/place_spec.rb#L69) the context is more descriptive.

Not a blocker, but something to consider as it would be better if things were consistent.

Reply:
Context changed to have a better name